server/routes/projectRoutes.js

```diff
@@ -0,0 +1,14 @@
+const express = require("express");
+const router = express();
+const {
+  getAllProjects,
+  getProjectsForUser,
+  createProjectForUser,
+} = require("../controllers/projectController");
+const auth = require("../middleware/authMiddleware");
+
+router.get("/", getAllProjects); // Get all projects (public)
+router.get("/:userId/", getProjectsForUser); // Get all projects for a specific user
```

Comment:
Will this be a public route? If we have public profiles then this is good

Reply:
Yea, i was thinking this route could be public for the profile page of a user. (for any public user to see all the projects of an entrepreneur)